scripts/e2e-compose-conflict.sh

```
@@ -65,13 +66,17 @@ prepare_conflict_provider_volume() {
 }
 
 prepare_conflict_review_driver() {
-  if [[ "$E2E_SCENARIO" != "conflict" && "$E2E_SCENARIO" != "conflict_queue" ]]; then
+  if [[ "$E2E_SCENARIO" != "conflict" && "$E2E_SCENARIO" != "conflict_queue" && !( "$E2E_SCENARIO" == "synchronous_write" && "${DENSE_MEM_E2E_WRITE_SLICE:-legacy}" == "conflict" ) ]]; then
```

Shellcheck (0.11.0)

[error] You are missing a required space here.

(SC1035)